src/cmd/kssh/kssh.go

```diff
@@ -235,13 +235,14 @@ func getConfig(botname string) (kssh.ConfigFile, error) {
 	// No specified bot and no default bot, fallback and load all the configs
 	configs, botnames, err := kssh.LoadConfigs()
 	if err != nil {
-		return empty, fmt.Errorf("Failed to load config file(s): %v\n", err)
+		return empty, fmt.Errorf("Failed to load config file(s): %v", err)
 	}
-	if len(configs) == 0 {
+	switch len(configs) {
+	case 0:
 		return empty, fmt.Errorf("Did not find any config files in KBFS (is `keybaseca service` running?)")
-	} else if len(configs) == 1 {
```

Comment:
up to you, but you can revert this if you take the client config settings. this rule was voted out

Reply:
/shrug I like that change so I'll leave it